EventVisualisation/Base/src/FileWatcher.cxx

```diff
@@ -0,0 +1,178 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file FileWatcher.h
+/// \brief Observing folder for created and removed files - preserving current
+/// \author julian.myrcha@cern.ch
+
+#include "EventVisualisationBase/FileWatcher.h"
+#include "FairLogger.h"
+
+#include "boost/filesystem/operations.hpp"
+#include "boost/filesystem/path.hpp"
+
+#include <iostream>
+#include <list>
+#include <sys/stat.h>
+using namespace std;
+
+namespace fs = boost::filesystem;
```

Comment:
please use std::filesystem, we had problems with boost::fs

Reply:
done